cmd/data/data.go

```diff
@@ -10,39 +11,47 @@ import (
 
 type Middleware func(http.HandlerFunc) http.HandlerFunc
 
+// ResultMetrics - type, that contains all gauge and counter metrics for representation them in HTML-format.
 type ResultMetrics struct {
-	GaugeMetrics   string
+	// GaugeMetrics contains all gauge metrics.
+	GaugeMetrics string
```

Comment:
```suggestion
	GaugeMetrics string // Сontains all gauge metrics.
```
Если комментарии не очень длинные, принято делать так, как в примере. IDE их даже специально форматируют. Не блокер.